specs/protocol/configurability.md

```diff
@@ -0,0 +1,71 @@
+# OP Stack Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Protocol Parameters](#protocol-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question.
+
+There are three categories of OP Stack configuration options:
+
+- **Protocol Parameters**: Parameters and properties of the chain that may
+  be set at genesis and fixed for the lifetime of the chain, or may be
+  changeable through a privileged account.
+- **Admin Roles**: These roles can upgrade contracts, change role owners,
+  or update protocol parameters. These are typically cold/multisig wallets not
+  used directly in day to day operations.
+- **Service Roles**: These roles are used to manage the day-to-day
+  operations of the chain, and therefore are often hot wallets.
+
+## Protocol Parameters
```

Comment:
"Superchain target" | Choice of cross-L2 configuration. May be omitted in isolated OP Stack deployments. Includes SuperchainConfig and ProtocolVersions contract addresses."

Or something to that extent should probably be part of the table.

Reply:
The table structure has changed slightly based on Mark's comments. Can you clarify what you mean with the above comment?

Reply:
I am suggesting to add an entry to the table

Reply:
Added superchain target into the consensus parameters table: https://github.com/ethereum-optimism/specs/pull/98/files#diff-f4cc226d6bd3074bf41cc708de428f532feff29c5c7f901b9cc5a5864f8a68b5R50